.github/workflows/ci_tests_dev.yaml

```diff
@@ -6,7 +6,7 @@ on:
   # push:
   #   branches: [ main ]
   pull_request:
-    types: [ready_for_review]
+    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
Need to revert this line before merging, right?

Reply:
Yep, done at 77a8a452e86412a7757e60d2b8fbfd6a9bdea83a